win-odbc64/sql/cli/sqlcli.h

```diff
@@ -425,6 +433,8 @@ enum SQLTYPE_CODE {
     SQLTYPECODE_BITVAR   = 15   /* not supported */
 };
 
+#define SQL_BOOLEAN			13
+
 /* #ifndef SQLDTCODE_DATE */
 /* specifies the type of datetime data type */
 enum SQLDATETIME_CODE {

```

Comment:
Is this latest version of sqlci.h from sql/cli? If not consider copying the latest version of sqlcli.h to this folder or creating a softlink to this file.